### PR TITLE
typescript + tables_manager

### PR DESCRIPTION
**Pull request by jonah-katz**

### CODE REVIEW

File: nodejs/src/test_data/game1.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        },
+        {
+          "player_id": 1,
+          "action": "big_blind",
+          "amount": 2
+        }
+      ]
+    ]
+  ]
+}
```
Comment (jonah-katz) [inline nodejs/src/test_data/game1.json:46]: @jacklieblich WIP but thinking something like this for the poker-table data model

File: nodejs/src/server.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import express from 'express';
```
Comment (jonah-katz) [inline nodejs/src/server.ts:1]: server.ts is the entry file for the backend

File: nodejs/src/classes/table.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  toJson(): any {
+    return {
+      small_blind: this.small_blind,
+      big_blind: this.big_blind,
+    }
+  }
+
+  getStateForClient() {
+    return this.toJson();
```
Comment (jonah-katz) [inline nodejs/src/classes/table.ts:19]: This function should accept a user id, and return a special state just for that user (ie. dont include other users' cards, only include "bet" button if its their turn, etc.)

**Review by jacklieblich — APPROVED** [review]
🤙

**Pull request merged**